modules/statics/src/coins.ts

```diff
@@ -7111,6 +7119,30 @@ export const coins = CoinMap.fromCoins([
     '0xe9e7cea3dedca5984780bafc599bd69add087d56',
     UnderlyingAsset['bsc:busd']
   ),
+  bscToken(
+    '2400c915-d610-45e7-bd7d-a028eb653b42',
+    'bsc:usdt',
+    'Tether USD',
+    18,
+    '0x55d398326f99059ff775485246999027b3197955',
+    UnderlyingAsset['bsc:usdt']
+  ),
+  bscToken(
+    '95296300-6d0b-4011-9028-53f54750d4b6',
+    'bsc:ksm',
+    'Kusama (Binance Pegged)',
```

Comment:
(Binance Pegged) doesn't look to be in the name: https://bscscan.com/address/0xe646c8fc9f507529b05fe0a825ae9beb1aad9f6b

Reply:
Is my first time with token integration and I used the names of the ticket, thanks for your comments. Is right "Kusama Token"?

Reply:
@mmcshinsky-bitgo since these are bitgo internal names, we want to denote that these are Binance pegged tokens; clients can get confused since there is the native L1 Kusama (KSM) that we dont want customer to accidently send in the wrong ones 
cc @evalenti-simtlix

Reply:
I'm good with whichever direction we go. Because they are here, they aren't internal, but are the name users will see if they were to pick them from a dropdown.